doc/ibm-cloud-vss.md

```diff
@@ -223,56 +237,32 @@ firewall-cmd --reload
 ```
 [More about firewalld](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-using_firewalls#sec-Getting_started_with_firewalld)
 
-#### Install DNS - based on dnsmasq 
-  * `yum install dnsmasq`
- 
-  * DHCP service must be turned off by adding the following entry in `/etc/dnsmasq.conf` for each interface: 
-```
-no-dhcp-interface=ens192
-```
-  * `systemctl enable dnsmasq.service` # so that dnsmasq will start after reboot
-  * `systemctl start dnsmasq.service`
-  
-`netstat -aunp` should show you that dnsmasq DNS is listening on 53.  It should NOT be listening on port 67 (DHCP)
+
+`netstat -aunp`
 ```
-netstat -aunp
  ...
 udp        0      0 0.0.0.0:53              0.0.0.0:*                           1202/dnsmasq         
 ```
 
 **Note: the following verifications need to be done after running the add_dns.sh script.( in the Update DNS step below)**
 
 * Verify that DNS lookups are good, and going to the right SERVER:
-```
-dig worker-00.myprefix.my.com
-...
+
+`dig worker-00.myprefix.my.com`
+```...
 ;; ANSWER SECTION:
 worker-00.myprefix.my.com. 0	IN	A	172.16.0.24
 ...
 SERVER: 127.0.0.1#53(127.0.0.1)
 ```
 
 * Verify that your SRV records are in DNS:
-```
-dig _etcd-server-ssl._tcp.myprefix.my.com SRV
```

Comment:
why remove this dig section?  it proves that the entries in dnsmasq.conf were added correctly

Reply:
deleted by mistake when I was trying to fix formatting I think. I can put back